padertorch/ops/losses/regression.py

```diff
@@ -49,45 +76,59 @@ def log_mse_loss(estimate: torch.Tensor, target: torch.Tensor,
             TasNet: A Dissecting Approach.” ArXiv:1911.08895
             [Cs, Eess], November 20, 2019.
             http://arxiv.org/abs/1911.08895.
+
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(0.9208)
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([0.0000, 0.9208])
     """
     # Use the PyTorch implementation for MSE, should be the fastest
     return _reduce(
-        F.mse_loss(estimate, target, reduce='none').mean(dim=-1).log10(),
-        reduce=reduce
+        F.mse_loss(estimate, target, reduction='none').mean(dim=-1).log10(),
```

Comment:
I think the mean should be a sum to be the same as the Demystifying TasNet paper

Reply:
I'd like to keep the mean, here, otherwise it is not a mean square error.
It has still the same gradient as the loss denoted in the Demystifying TasNet paper.
So this change has no large impact. (Moving the 1/T outside the log it is only a bias)

I don't know how the mean operation is implemented,
but in theory the mean operation can be more stable (e.g. avoid overflow).
(Probably it relies on sum and is not more stable)